nuxeo-core/nuxeo-core-api/src/main/java/org/nuxeo/ecm/core/api/DocumentExistsException.java

```diff
@@ -19,6 +19,10 @@
  */
 package org.nuxeo.ecm.core.api;
 
+import javax.servlet.http.HttpServletResponse;
+
+import static javax.servlet.http.HttpServletResponse.SC_CONFLICT;
```

Comment:
I believe the Nuxeo code format has the `import static` before other imports

Reply:
:+1: I'm planning to do a big format commit on all updated files!